packages/http/src/index.ts

```diff
@@ -20,9 +20,7 @@ import { validator } from './validator';
 
 const createInstance = <LoaderInput>(overrides?: TPrismHttpComponents<LoaderInput>) => {
   return factory<IHttpOperation, IHttpRequest, IHttpResponse, IHttpConfig, LoaderInput>({
-    config: {
-      mock: true,
-    },
+    config: {},
```

Comment:
I don't know why it's broken, but look at line 29 - `overrides` should override this configuration.
E.g. if someone passes their own `mock` in the CLI they should be able to change the settings **globally**.
I'm very surprised this doesn't work. Do we have unit/int tests confirming that?